src/test/scala/bloop/integrations/gradle/ConfigGenerationSuite.scala

```diff
@@ -3632,6 +3632,7 @@ abstract class ConfigGenerationSuite extends BaseConfigSuite {
         |
         |configurations {
         |    scalaCompilerPlugin
+        |    bloopConfig.extendsFrom(scalaCompilerPlugin)
```

Comment:
This isn't needed is it?

Reply:
It actually is, because as I say in the description this configuration is user-defined. If it was `scalaCompilerPlugins` it would be extended on by default. But this one is user-defined, so users need to extend from it if they want the artifacts to be exported to the resolution module.
The great thing about this approach is that it allows users to configure which artifacts are exported, which before wasn't possible.

Reply:
OK. But if I run this test with or without that `extendsFrom` line, `semanticdb-scalac_2.12.8` is always present in the `resolution/modules`.  Is that what the difference is supposed to be?  I'm using Gradle 8.4 and jdk 21.

Reply:
I have no idea why, but this test was failing for me. This was the way to make it work.
It might or might not be needed, I'm just doing `sbt +test` to check everything works.
Does that work for you without this `extendsFrom`?

Reply:
Yes - without that line the test still passes for me - I can't get it to fail no matter what Gradle and JDK version I use.
It would make sense to me that the line isn't needed as in `BloopPlugin#extendCompatibleConfigurationAfterEvaluate` you extend every `project.getConfigurations`.  Wouldn't that include the user-defined ones?  And before this PR the code just iterated through `project.getConfigurations`.  I think most projects would want all configs included by default wouldn't they?